lib/nexthop_group.c

```diff
@@ -706,6 +706,30 @@ DEFPY(no_nexthop_group_resilience,
 	return CMD_SUCCESS;
 }
 
+DEFPY(nexthop_group_recursive,
```

Comment:
Can we document this?

Reply:
doh! yes, of course
this is really just a draft, so I know there are still some rough edges...

Reply:
pushed it...